htmlcss/task2/header.css

```diff
@@ -0,0 +1,149 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'SourceSansPro-Regular', sans-serif;
+}
+
+body {
+    background:#26272f url(images/Background.png) no-repeat; 
+}
+
+A:link { 
+    text-decoration: none; 
+     
+} 
+
+A:visited { 
+    text-decoration: none;
+} 
+    
+ A:active {
```

Comment:
Do not use html tags in css, it's bad practice according to bem. Same applies to header